case-lib/logging_ctl.sh

```diff
@@ -28,9 +28,16 @@ _func_log_cmd()
     # writing functions
     shopt -s expand_aliases
     if [ "X1" ]; then
+        local ppcmd=$(ps -p $PPID -o cmd --noheader|awk '{print $2;}') ext_message=""
```

Comment:
Where is PPID defined? 
awk '{print $2;}' in my environment, it is the parameter, not the cmdline if I run it directly locally.

Reply:
PPID define by bash  
`PPID   The process ID of the shell's parent.  This variable is readonly.`

Reply:
Oh, right, PPID means the parent PID. Please ingore this comment